tests/test_tiktoken.py

```diff
@@ -259,3 +279,21 @@ def test_additional_special_tokens(model_name: Optional[str],
 
     assert encoded_outputs[0] == wrapped_tokenizer.vocab_size
     assert len(encoded_outputs) == 2
+
+
+@pytest.mark.parametrize('model_name,encoding_name',
+                         MODEL_ENCODING_NAME_PARAMETRIZATION)
+def test_chat_formatting(model_name: Optional[str],
+                         encoding_name: Optional[str], tmp_path: pathlib.Path):
+    special_tokens_to_add = ['<|im_start|>', '<im_end>']
+    wrapped_tokenizer, _, _ = get_tokenizers_for_testing(
+        model_name,
+        encoding_name,
+        tmp_path,
+        add_bos_token=False,
+        add_eos_token=False,
+        additional_special_tokens=special_tokens_to_add)
+    for i, dict_chats in enumerate(MULTI_TURN_CHAT_ML):
```

Comment:
looks like this is a loop over a list of length 1? did you intend for it to have more items? or should it not be a list?

Reply:
its so that we can easily add more cases in the future when we have default system prompts and various forms of multi turns